recipes/Media/ASR/seq2seq/hparams/train_with_wav2vec.yaml

```diff
@@ -0,0 +1,156 @@
+# ################################
+# Model: Wav2Vec + DNN + CTC + Softmax
+# Authors modifier: Gaelle Laperriere
+# ################################
+
+# ------ Paths and parameters
+
+# Seed needs to be set at top of yaml, before objects with parameters are made.
+seed: 4242
+__set_seed: !!python/object/apply:torch.manual_seed [!ref <seed>]
+output_folder: !ref results/media_ASR_wav2vec/<seed>
+cer_file_test: !ref <output_folder>/cer_test.txt
+ctc_file_test: !ref <output_folder>/ctc_test.txt
+save_folder: !ref <output_folder>/save
+train_log: !ref <output_folder>/train_log.txt
+
+# See https://github.com/pytorch/fairseq/blob/main/examples/wav2vec/README.md
+# for Wav2Vec models and https://huggingface.co/LeBenchmark for French ones.
+wav2vec_url: LeBenchmark/wav2vec2-FR-3K-large
+
+# Data files:
+data_folder: !PLACEHOLDER
+csv_train: !ref <data_folder>/train.csv
+csv_valid: !ref <data_folder>/dev.csv
+csv_test: !ref <data_folder>/test.csv
+
+# Data parameters:
+# With data_parallel batch_size is split into N jobs.
+# With DDP batch_size is multiplied by N jobs.
+batch_size: 4
+test_batch_size: 2
+# We remove utterances longer than 90s in the train/dev/test sets as
+# longer sentences certainly correspond to "open microphones".
+avoid_if_longer_than: 90.0
+avoid_if_smaller_than: 0.0
+dataloader_options:
+    batch_size: !ref <batch_size>
+    num_workers: 3
+    shuffle: True
+test_dataloader_options:
+    batch_size: !ref <test_batch_size>
+    num_workers: 3
+
+# Feature parameters:
+sample_rate: 16000
+feats_dim: 1024
+
+# Training parameters:
+number_of_epochs: 77
+lr: 1
+lr_wav2vec: 0.0001
+annealing_factor: 0.8
+annealing_factor_wav2vec: 0.9
+improvement_threshold: 0.0025
+improvement_threshold_wav2vec: 0.0025
+patient: 0
+patient_wav2vec: 0
+sorting: random
+
+# Model parameters:
+activation: !name:torch.nn.LeakyReLU
+dropout: 0.15
+cnn_blocks: 0
```

Comment:
remove cnn and rnn block from the yaml if they are not used.

Reply:
Correction done.